docker-compose.yml

```diff
@@ -62,18 +39,10 @@ services:
     links:
       - db
     networks:
-      - new
-    image: node:18.12.1-alpine
-    working_dir: /target
-    volumes:
-      - .:/target
-      - status:/in
-    command: "sh -c 'until [[ -f /in/db-init-done && -d /target/node_modules ]]; do sleep 1; done; yarn migrate; yarn prisma db seed'"
```

Comment:
We're losing all the important findings @RageZBla made and I think we need to back up and confirm what the root cause is before spending time rewriting other people's work. Let's get on a call